scripts/docker/test.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+function test() {
+  local package="${1:?Provide a package}"
+  local sub_package="${2:-}"
+
+  export DIR=${package}
+  . <(/ci/shared/helpers/extract-default-params-for-task.bash /ci/shared/tasks/run-bin-test/linux.yml)
```

Comment:
Is this call needed since we're passing DEFAULT_PARAMS to the run-bin-test script on line 13?

Reply:
This will load default values for params that are defined globally e.g. verify_go for VERIFICATIONS. The intention here is that the script that would run inside of the docker container would be exactly identical to what would get executed in concourse. In other words, by fetching the latest `tas-runtime-postgres` image and running the tests in docker we would see go versions mismatch between the release and docker container the same way as it would happen on concourse.

`DEFAULT_PARAMS`  is convention by which a release e.g. routing-release would defines default values for params for which is either not defined or would like to be overridden. This is only needed for run-bin-test or build-binaries since those would be the most common thing a contributor would run.